exercises/Anne-Hyacinthe/answers.js

```diff
@@ -0,0 +1,130 @@
+// Question 1
+function greet(name) {
+ return (`Good morning, ${name}!`);
+}
+
+// Question 2
+function logOddNumbers() {
+  for(i = 1; i <= 99; i += 1)
+    if (i % 2 !== 0){
+      console.log(i);
+    }
+}
+
+// Question 3
+function area(length, width) {
+ return length * width;
+  
+}
+
+// Question 4
+function otherAngle(angle1, angle2) {
+  let MissingAngle = 180 - (angle1 + angle2)
+  return MissingAngle
+}
+
+// Question 5
+function diameterFromArea(area) {
+   let radius2;
+  radius2 = area / Math.PI ;
+  let x;
+  x= Math.sqrt(radius2)
+  let diameter = x * 2 ;
+  return diameter;
+}
+
+// Question 6
+function totalPrice(cost, tax) {
+  percent = cost / 100;
+  per = tax * percent;
+  total = per + cost;
+  return(total);
+}
+
+// Question 7
+function shortLongShort(e,y) {
+   if (e.length > y.length) {
+    return (y+e+y)
+  } else if (y.length > e.length){
+    return (e+y+e)
+  }
+  
+  
+}
+
+// Question 8
+function describeAge(age) {
```

Comment:
Too many extra conditional checks here. Take advantage of the fact that `if... else` statements execute from top to bottom. Rewrite this without using any `&&`s.